exo-setup/src/docker-setup.ls

```diff
@@ -44,6 +46,19 @@ class DockerSetup extends EventEmitter
       no
 
 
+  _setup-external-service: (done) ~>
+    if @config.docker-image
+      image = @config.docker-image |> (.split '/')
+      new ObservableProcess((DockerHelper.get-pull-command author: image[0], name: image[1]),
+                            stdout: {@write}
+                            stderr: {@write})
+        ..on 'ended', (exit-code) ~> done!
```

Comment:
You should check the exit code here, and display at least an error message when something went wrong. Or bubble the error up and handle it higher up in the call chain via `..on 'ended', done`.